sets/core/step.py

```diff
@@ -37,3 +44,134 @@ def directory(cls, prefix=None):
         directory = os.path.expanduser(os.path.join(prefix, name))
         utility.ensure_directory(directory)
         return directory
+
+    @classmethod
+    def _parse_hdf5path(cls, path):
+        """ Splits hdf5 path into file and column name
```

Comment:
- Don't call this column (maybe *inner file*?)
- Don't write docstrings in third person (`Split ...` vs `Splits ...`)
- Break before description line.
- Use Google-style docstrings (missing colon after *Return* and `name (type): description`)